# FeetOutputMsg.msg

time stamp
FootOutputMsg leftFoot
FootOutputMsg rightFoot